src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25943.cs

```diff
@@ -12,27 +12,23 @@ public Bugzilla25943(TestDevice testDevice) : base(testDevice)
 
 		public override string Issue => "[Android] TapGestureRecognizer does not work with a nested StackLayout";
 
-		/*	
+
 		const string InnerLayout = "innerlayout";
 		const string OuterLayout = "outerlayout";
 		const string Success = "Success";
 
 		[Test]
-		[Category(UITestCategories.LifeCycle)]
+		[Category(UITestCategories.Gestures)]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnAndroidWhenRunningOnXamarinUITest]
-		[FailsOnIOSWhenRunningOnXamarinUITest]
-		[FailsOnMacWhenRunningOnXamarinUITest]
 		public void VerifyNestedStacklayoutTapsBubble()
 		{
 			App.WaitForElement(InnerLayout);
 			App.Tap(InnerLayout);
```

Comment:
Methods like `WaitForElement` or `Tap` for example, search first by AutomationId and if don't find the element, make a query using XPath searching by content (text etc). In principle, it shouldn't be necessary to modify the test example by adding AutomationIds for it to work.

On the other hand, using AutomationId is more efficient than queries, so it's a good practice.